css/index.css

```diff
@@ -88,29 +112,145 @@ p {
 .middle-content {
     margin-bottom: 20px;
     border-bottom: 1px dashed black;
+    width:100%;
+    flex-wrap: wrap;
 }
 
 .middle-content h2 {
     padding: 0 2%;
     margin-bottom: 0;
 }
+.middle-content .box1{
+    background:teal;
+}
+.middle-content .box2{
+    background:gold;
+}
+.middle-content .box3{
+    background:cadetblue;
+}
+.middle-content .box4{
+    background:coral;
+}
+.middle-content .box5{
+    background:crimson;
+}
+.middle-content .box6{
+    background:forestgreen;
+}
+.middle-content .box7{
+    background:darkorchid;
+}
+.middle-content .box8{
+    background:hotpink;
+}
+.middle-content .box9{
+    background:indigo;
+}
+.middle-content .box10{
+    background:dodgerblue;
+}
 
 .middle-content .boxes {
     display: flex;
     flex-wrap: wrap;
     justify-content: space-evenly;
+    box-sizing: content-box;
+    width: 100%;
+    padding:2%;
 }
 
-.middle-content .boxes .box {
+.middle-content .boxes .box1{
+    width: 12.5%;
+    height: 100px;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+.middle-content .boxes .box2{
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
+.middle-content .boxes .box3{
```

Comment:
While this gets the job done, it isn't very efficient. since all boxes have similar properties (height, width, margin...) it would be best to have one class with the basic styling properties and then use a CSS Selector to target each box just to change the color